### DISCUSSION TITLE
Stock application

### DISCUSSION THREAD

**Discussion by Villevalld** [Budibase/budibase#5600] [Help] [answered]
Hello Everyone!

I’m about to create small stock application. So I have prepared tables as below

**Item**

- Part number
- Description	

**Stock**

- Item.Part Number
- Amount

**Order**

- Order number (AutoID)
- Date

**OrderItems**

- Order.Order number
- Item.Part Number
- Amount

Now the Issue: 
Assumed that the tables Item and Stock are prefilled. Then I create new Order. After save the app navigate to created order. In this view I additionally see the table „OrderItems“ with related items to the order. My issue is to add new Item (row) to this table. In particularly when I create new view for OrderItems I must preselect the field Order.Order number. I’ve tried to do that with url like /orderItems/new/:orderid and then set the default value for the field to {{ URL.orderid }} but this dis not worked for me. I assume it is because the field is a relationship and not just a value. Any Idea how to solve this?

Thanks!

**Comment by melohagan** [comment, marked as answer]
Hi @Villevalld 

If you want to save a relationship, you will need to pass the **_id** of the order row, instead of an AutoID.

**Reply by Villevalld** [reply]
Hi @melohagan Thanks a lot! It worked for me!
